CombineUnloadAIDriver.lua

```diff
@@ -59,6 +59,9 @@ CombineUnloadAIDriver.minDistanceFromReversingChopper = 10
 CombineUnloadAIDriver.minDistanceFromWideTurnChopper = 5
 CombineUnloadAIDriver.safeManeuveringDistance = 30 -- distance to keep from a combine not ready to unload
 CombineUnloadAIDriver.pathfindingRange = 5 -- won't do pathfinding if target is closer than this
+CombineUnloadAIDriver.maxFruitPercent = 10
```

Comment:
This way these are class (static) variables. Since we are changing them, they should be instance variables defined in init()
Also, a resetPathfinder() or something would make sense to avoid repeatedly setting the default values in init() and on pathfinding end.